.gitignore

```diff
@@ -25,4 +25,4 @@ public/*
 
 coverage
 lib
-artifacts
+artifacts
```

Comment:
our codebase required line return to avoid those dummy diff (probably a settings in your IDE to set up)